scripts/auto_backtester/backtester.js

```diff
@@ -79,7 +79,7 @@ let runCommand = (strategy, cb) => {
     speed: `--baseline_periods=${strategy.baseline_periods} --trigger_factor=${strategy.trigger_factor}`,
     trend_ema: `--trend_ema=${strategy.trend_ema} --oversold_rsi=${strategy.oversold_rsi} --oversold_rsi_periods=${strategy.oversold_rsi_periods} --neutral_rate=${strategy.neutral_rate}`
   };
-  let command = `node zenbot sim ${simArgs} ${strategyArgs[strategyName]} --period=${strategy.period}  --min_periods=${strategy.min_periods}`;
+  let command = `zenbot sim ${simArgs} ${strategyArgs[strategyName]} --period=${strategy.period}  --min_periods=${strategy.min_periods}`;
```

Comment:
Well, looks the prefix `node` was used to fix Windows platform issue and it was not patched by me(I don't use Windows any more)...
https://github.com/carlos8f/zenbot/pull/346

Anyway, this will break backtester.js under Linux and looks could not solve the Windows issue, too. As stackoverflow discussed, `npm link` should already make a `zenbot.cmd` wrapper for users, so could someone usubg Windows to test this problem again to ensure if `node zenbot` is necessary.
https://stackoverflow.com/questions/10396305/npm-package-bin-script-for-windows